CONTRIBUTING.md

```diff
@@ -104,6 +104,8 @@ Some tests require a Windows VM or Linux containers. For Linux tests (recommende
 
 Additional tests may run in a local environment and require certain shell capabilities. For example tests that require a Bash-like environment are tagged with `:bash` in rspec.
 
+Some tests will also require that the [bundled modules](#contributing-to-bundled-modules) described above are installed. Ensure the modules are installed for testing with `bundle exec r10k puppetfile install .`.
```

Comment:
```suggestion
Some tests will also require that the [bundled modules](#contributing-to-bundled-modules) described above are installed. Ensure the modules are installed for testing with
```
bundle exec r10k puppetfile install
```
```

Reply:
Ugh this formatting is weird. Basically move it into a newline :p

Reply:
Ha, yeah that is a weird one. Updated.